### ISSUE TITLE
Slow booting time

### ISSUE THREAD

**Issue by dgironella** [owenvoke/blade-fontawesome#43]
## Description

I find in one of our servers that if install this package laravel have a big slow booting. More than 4 seconds. If we remove this package all work correctly.

Possible some app provider register? Don't know, or need some php library version. Not errors on logs. Only slow booting.

## Your environment

PHP 8.1.8
Laravel 9

**Comment by anilken** [comment]
cache your files using the `php artisan icons:cache` command it will work fast

**Comment by Wit3** [comment]
> Cache your files using the `php artisan icons:cache` command it will work fast

`php artisan icons:cache`

doesn't solve the problem at all, my boot time w/o blade fontawesome is 70 ms and with this package ( cached ) is 400 ms

**Comment by vlados** [comment]
I can confirm. On a new installation the boot time jumps from ~140 to ~280-300ms

**Comment by owenvoke** [comment]
Yeah, there are definitely issues with the way Blade UI Icons work. 🤷🏻 I think it's just because there are so many files, not really much we can do.

If you find a solution, feel free to PR it. 👍🏻

**Comment by Excelautomation** [comment]
`php artisan icons:cache` worked for me. I had 27 seconds loadtime before caching.
